src/vip/batch_address_test_tool/processor.clj

```diff
@@ -70,11 +71,23 @@
 
 (def retrieve-polling-locations (->pass-through retrieve-polling-locations*))
 
-(defn calculate-match*
+(defn calculate-scores*
   [ctx]
-  ctx)
+  (let [addresses (:addresses ctx)
+        scores (map match/calculate-score addresses)
+        merged (map #(assoc %1 :score %2) addresses scores)]
+    (assoc ctx :addresses merged)))
+
+(def calculate-scores (->pass-through calculate-scores*))
+
+(defn calculate-matches*
+  [ctx]
+  (let [addresses (:addresses ctx)
+        matches (map match/calculate-match addresses)
+        merged (map #(assoc %1 :match %2) addresses matches)]
+    (assoc ctx :addresses merged)))
 
-(def calculate-match (->pass-through calculate-match*))
+(def calculate-matches (->pass-through calculate-matches*))
```

Comment:
I think if we were doing this *a lot* I'd maybe re-write it, but I think since it's just two instances it's clearer the way we originally wrote it.  @tank157 and I discussed doing something like what you suggest and decided to leave it but we might consider it if other similar pieces were going to be added in the future (which I don't think so but who knows).